content/articles/what-happens-if-i-stop-paying.markdown

```diff
@@ -0,0 +1,16 @@
+---
+title: What happens if I stop paying for my DNSimple subscription?
+excerpt: Description of what happens if an account is unsubscribed. 
+categories:
+ Account
+---
+
+# What happens if I stop paying for my DNSimple subscription?
+
+After a certain period of time, an account is cancelled after payment cannot be collected. What happens if such a thing were to happen depends on the current configuration of your delegation. 
```

Comment:
No idea what I was thinking when I wrote that. Reading it now makes my brain hurt. Fixing immediately, thanks @jcaudle.